sources/SilkTouch/SilkTouch/Mods/MixKhronosData.cs

```diff
@@ -295,143 +337,122 @@ .. currentConfig.NonStandardExtensionNomenclature
 
             job.TypeMap.TryAdd(type, baseType);
         }
+
+        // Add mappings from Flags storage types to FlagBits enums types
+        // We want Flags as the output, but in order to generate everything correctly,
+        // we need to first map Flags to FlagBits, then rename FlagBits back to Flags (this is done in the rewriters).
+        // Without this, ClangSharp will output the Flags types as the integral storage type instead of the enum
+        foreach (var typeElement in xml.Elements("registry").Elements("types").Elements("type"))
+        {
+            var typedef = typeElement.Element("type")?.Value;
+            if (typedef != "VkFlags" && typedef != "VkFlags64")
```

Comment:
Not sure this should be hardcoded. This should probably be configurable with these as sensible defaults.